spec/requests/admin/admin_sessions_acceptance_spec.rb

```diff
@@ -6,7 +6,7 @@
     Admin.create!(email: 'hello@withassociates.com', password: '123456')
     StandardTree.build_minimal
 
-    visit '/admin/sign_in'
+    visit '/admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.